tools/fontbakery-metadata-vs-api.py

```diff
@@ -129,26 +130,26 @@ def getVariantName(item):
                 repo_md5 = hashlib.md5(open(os.path.join(dirpath, repoFileName), 'rb').read()).hexdigest()
 
                 if repo_md5 == google_md5:
-                    log_messages.append([variant, 'OK', '{} in sync'.format(repoFileName)])
+                    log_messages.append([variant, 'OK', '"{}" in sync'.format(repoFileName)])
                 else:
-                    log_messages.append([variant, 'ER', '{}: Checksum mismatch: File in API does not match file in repository'.format(repoFileName)])
+                    log_messages.append([variant, 'ER', '"{}" checksum mismatch, file in API does not match file in git'.format(repoFileName)])
 
             except ValueError:
-                log_messages.append([variant, 'ER', 'Available in API but not in repository'])
+                log_messages.append([variant, 'ER', '"{}" available in git but not in API'.format(font.filename)])
```

Comment:
I was trying to address the way that 

    ER: Available in repository but not in API

lacks context to tell you which family